oci/layer/unpack.go

```diff
@@ -226,7 +234,7 @@ func UnpackManifest(ctx context.Context, engine cas.Engine, bundle string, manif
 		// in the layer.
 		_, _ = io.Copy(ioutil.Discard, layer)
 		// XXX: Is it possible this breaks in the error path?
-		layerGzip.Close()
+		layerRaw.Close()
```

Comment:
This should be `layerData`, assuming you wanted to have the same semantics.

Reply:
Arg, yeah, thanks. Just pushed a fix.